### PR TITLE
Standup update

### PR DESCRIPTION
**Pull request by JimMadge**

### CODE REVIEW

File: weekly-standup-template.md [hunk 1]
```diff
@@ -76,33 +76,13 @@ For example, this may be an emergent task, providing support or expertise to oth
   - Draft PR to fix bug [created](https://…)
 -->
 
-##### [Identify and implement core IAC changes](https://github.com/alan-turing-institute/data-safe-haven-team/issues/28)
+##### [Release 5.1.0](https://github.com/alan-turing-institute/data-safe-haven/milestone/28)
```
Comment (jemrobinson) [inline weekly-standup-template.md:79]: Should this be "next release" or "current release"?

**Review by jemrobinson — APPROVED** [review]
Can we add a story about TRESA/production work?

File: weekly-standup-template.md (hunk 1 above)
Comment (JimMadge) [reply weekly-standup-template.md:79]: I was planning to update it when a minor release is made.

I want to try and make the point that there are two acceptable types of tasks for developers to pick up. Hotfix issues which will result in a patch release, or issues on the milestone for the next minor/major release.

I wanted to make it clear that means 5.1.0, not 5.2.0 or 6.*.

Do you think there is a better way to do that?

Comment (jemrobinson) [reply weekly-standup-template.md:79]: "Next patch release" maybe? Just saves changing the template each time we release :)

Comment (JimMadge) [reply weekly-standup-template.md:79]: I'll need to change the link each time anyway (unless we remove that).

**Pull request merged**